### PR TITLE
User can view a specific Trail

### PR DESCRIPTION
**Pull request by jonas-masharqa**
[PT-Link](https://www.pivotaltracker.com/story/show/169667928)

## PR Contains
```
As a user
In order to read more about a trail
I would like to have a page to view the specific trail
```

## PR Content
- Integration test Happy & Sad paths
- Fixtures
- Refactoring of error handling
- `SpecificTrail` component + Path/NavLink
- Styling for displaying Specific trail

## Screenshots
Specifictrail

### CODE REVIEW

**Review by piavW — APPROVED** [review]

**Review by lucamarial — APPROVED** [review]

File: src/Components/LandingPage.jsx [hunk 1]
```diff
@@ -2,37 +2,32 @@ import React, { Component } from 'react'
...
-      errorMessage: error
-    })
-  }
-
-  async getTrailsData() {
-    let fetch = await getTrails()
-    if (fetch.error) {
-      this.setErrorMessage(fetch.error)
+  async componentDidMount() {
+    debugger
```
Comment (AlexSaxena) [inline src/Components/LandingPage.jsx]: Please remove debugger

File: src/Modules/trailsData.js [hunk 2]
```diff
@@ -3,13 +3,21 @@ import axios from 'axios'
 const apiUrl = 'http://localhost:3000/v1/'
 
 const getTrails = async () => {
+  debugger
```
Comment (AlexSaxena) [inline src/Modules/trailsData.js]: Please remove debugger

**Review by AlexSaxena — APPROVED** [review]

**Review by DIzosimov — APPROVED** [review]

File: src/Components/LandingPage.jsx (hunk 1 above)
Comment (jonas-masharqa) [reply src/Components/LandingPage.jsx]: Done!

File: src/Modules/trailsData.js (hunk 2 above)
Comment (jonas-masharqa) [reply src/Modules/trailsData.js]: Done!

**Pull request merged**